### PR TITLE
Add Direct Banking to Connectivity Apps

### PR DESCRIPTION
**Pull request by RSchoesterIdyn**
Added Appsource registration for Direct Banking NL from Idyn B.V.

### CODE REVIEW

**Comment by RSchoesterIdyn** [comment]
@microsoft-github-policy-service agree company="IDYN B.V."

File: Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al [hunk 1]
```diff
@@ -165,6 +166,34 @@ codeunit 20352 "Connectivity App Definitions"
...
+    begin
+        /***************************************************
+            Add app 'Direct Banking' to NL
+        ***************************************************/
+
+        AppId := '7d5b57c9-71d8-47f0-85b8-7a08066f7d2b';
+        AppName := 'Direct Banking NL';
+        AppPublisher := 'IDYN B.V.';
+        AppDescription := 'Integrate ABN, Rabo, ING, Triodos, Knab, ASN, SNS, etc. with Microsoft Dynamics 365 Business Central.';
+        AppProviderSupportURL := 'https://help.idyn.nl/directbanking';
```
Comment (sorenfriisalexandersen) [inline Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al:189]: @RSchoesterIdyn This URL must point to a page where the user immediately sees the list of banks they can connect to through your app. With such page available, the list of supported banks is only one click away from BC. Thanks.

**Review by sorenfriisalexandersen — CHANGES_REQUESTED** [review]

File: Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al (hunk 1 above)
Comment (RSchoesterIdyn) [reply Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al]: Dear @sorenfriisalexandersen: 
I have updated the url for the support page.

However, listing the banks directly on our support page is not possible.
Our payment provider has very frequent updates of their list of supported banks and our support page doesn't allow to embed external pages.
Instead, I added a direct link to the website of our payment provider, which has the full list of supported banks.
So, it's two clicks now.

**Review by sorenfriisalexandersen — APPROVED** [review]

**Comment by haoranpb** [comment]
FYI, I will be move the code to line 227, so it's easier to tell the order when an app is added.

Will do it when resolving conflicts, since there is another Connectivity Apps PR

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 52045.

**Comment by JesperSchulz** [comment]
Availability update: We will publish a fix for this issue in the next update for release 21.

Build ID to track: 52214.

**Pull request merged**